.github/ISSUE_TEMPLATE/feature-request-.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Would you mind renaming to `feature_request.md` just for consistency with other repos?